README.md

```diff
@@ -77,6 +77,9 @@ In Linux it's typically located in /etc/mysql
 
 Restart WAMPP/XAMPP Server.
 
+(XAMPP)
```

Comment:
Some thoughts. :)
* 'parametery' -> 'parameter'
* 'in my.ini file' -> 'in the my.ini file'
* remove space after /phpmyadmin/
* 'search for "sql mode" and' -> 'search for "sql mode," and'
* maybe just 'set to ""'?

Reply:
Thanks
I've just corrected it  :)